x/staking/migrations/v5/keys.go

```diff
@@ -15,9 +15,10 @@ const (
 )
 
 var (
-	DelegationKey           = []byte{0x31} // key for a delegation
-	HistoricalInfoKey       = []byte{0x50} // prefix for the historical info
-	DelegationByValIndexKey = []byte{0x71} // key for delegations by a validator
+	DelegationKey                             = []byte{0x31} // key for a delegation
+	HistoricalInfoKey                         = []byte{0x50} // prefix for the historical info
+	DelegationByValIndexKey                   = []byte{0x71} // key for delegations by a validator
+	NextMigrateDelegationsByValidatorIndexKey = []byte{0x81} // key used to migrate to the new validator index
```

Comment:
I think this is safe because we eventually will not need this key and we will remove it from the store right? Was initially worried about sdk eventually using this prefix for something else.

With that being said, can we add a TODO to purge this key being stored after v26? Alternatively, we could have the migration delete the key once all stores have been migrated so we don't need to worry about it anymore.

Reply:
The migration will delete the key when it's completed

see: https://github.com/osmosis-labs/cosmos-sdk/pull/622/files#diff-273579e75b5cd0cebdf3f61d2784e9250831661eef8fc7377f5585329db29e4dR56-R59